lib/hoosegow/docker.rb

```diff
@@ -126,7 +126,11 @@ def stop_container
     # Returns response body or nil if no container was started.
     def delete_container
       return unless @container
-      @container.delete
+      begin
+        @container.delete
+      rescue Docker::Error::ServerError => e
+        raise e unless e.end_with? "device or resource busy"
+      end
```

Comment:
The `begin` and `end` aren't necessary, since this method is so short.

Do you know which other errors delete might raise? I think it'd be fine to swallow any `Docker::Error::ServerError`. It might be nice to log them.